### PR TITLE
Added pt-BR translation

### PR DESCRIPTION
**Pull request by halcance**
### Description

Translated the tags to pt-BR

### Motivation

I'm a member of SUMO and L10n Brazil and the changes will help my community.

### Related issues and pull requests

Relates to #8749
https://github.com/mdn/translated-content/issues/8749#issue-1383308373

### CODE REVIEW

**Comment by cw118** [comment]
@mdn/yari-content-pt-br could someone review this please?

**Review by wbamberg — COMMENTED** [review]
As I commented in https://github.com/mdn/content/pull/21245, alphabetical ordering would be nice.

**Comment by halcance** [comment]
Tagging you back @bsmth I didn't mean to remove you

Github trolled me

File: files/jsondata/L10n-CSS.json [hunk 1]
```diff
@@ -1,73 +1,82 @@
 {
   "css_ref_title": {
+    "es": "Referencia CSS",
     "en-US": "CSS Reference",
```
Comment (wbamberg) [inline files/jsondata/L10n-CSS.json:4]:
```suggestion
    "en-US": "CSS Reference",
    "es": "Referencia CSS",
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @halcance !

**Pull request merged**